client/src/styles/datetime.css

```diff
@@ -0,0 +1,80 @@
+.react-datepicker-wrapper {
```

Comment:
I have adjusted the height of the time dropdown with the screen having max-height: 670px (In my developer tool, iphone 6/7/8's size is 375x667).
I also added `phoneMini` size (max-width: 375px) in `util.js` and used it to adjust the font-size of the selectors in shorter screen, so that the shape of the whole selector doesn't change.

Reply:
Fantastic!